src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -36,10 +36,10 @@
 
   private MotorType DRIVE_MOTOR_TYPE = MotorType.kBrushless;
 
-  private boolean gyroDisabled = false;
+  private boolean gyroDisabled = true;
 
   public Drive(Gyroscope gyroscope) {
-    this.leftGearbox = new Gearbox(new Encoder(0, 1), new CANSparkMax(LEFT_BACK_CAN_ID, DRIVE_MOTOR_TYPE),
+    this.leftGearbox = new Gearbox(new Encoder(7, 8, true), new CANSparkMax(LEFT_BACK_CAN_ID, DRIVE_MOTOR_TYPE),
```

Comment:
```suggestion
    this.leftGearbox = new Gearbox(new Encoder(7, 8), new CANSparkMax(LEFT_BACK_CAN_ID, DRIVE_MOTOR_TYPE),
```

Let's not invert encoders here. I added this for testing.